content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Spring Boot
+linkTitle: Spring Boot
+weight: 30
+description: Spring instrumentation for OpenTelemetry Java
+javaInstrumentationVersion: 1.31.0
+cSpell:ignore: springboot autoconfigure 
+---
+
+You can automatically instrument a [Spring Boot](https://spring.io/projects/spring-boot) application using the [OpenTelemetry Java agent](../automatic/) that uses byte code instrumentation.
+
+The OpenTelemetry Spring Boot Starter is another way to instrument your Spring Boot application. It is a [Spring Boot starter](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.build-systems.starters).
+
+The OpenTelemetry starter is compatible with Spring Boot 2.0, Spring Boot 3.0 and Spring native.
+
+## Getting Started
+
+Add the dependency given below to enable the OpenTelemetry starter.
+
+The OpenTelemetry starter uses OpenTelemetry [Spring Boot auto-configurations](https://docs.spring.io/spring-boot/docs/current/reference/html/using.html#using.auto-configuration).
+
+You can look at [this](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/spring/spring-boot-autoconfigure/README.md#features) to know the supported libraries and features of the OpenTelemetry auto-configurations.
+
+### Maven
+
+Add the following dependencies to your `pom.xml` file:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-spring-boot-starter</artifactId>
+    <version>{{% param javaInstrumentationVersion %}}</version>
+  </dependency>
+</dependencies>
+```
+
+### Gradle
+
+Add the following dependencies to your gradle.build file:
+
+```groovy
+implementation("io.opentelemetry.instrumentation:opentelemetry-spring-boot-starter:{{% param javaInstrumentationVersion %}}")
+```
```

Comment:
Use tabs for the alternatives "Maven" and "Gradle"

Reply:
@chalin I am not sure to follow. In my last commit I have aligned  this Gradle example with another one found on this repository.

Reply:
It should be fine my new commit